photonjs/README.md

```diff
@@ -192,20 +198,129 @@ await photon.user.findMany({
 })
 
 // Get first of many
-const user: User | null = await photon.user
-  .findMany({ where: { lastName: 'Doe' } })
-  .first()
+const user: User | null = await photon.user.findMany({ where: { lastName: 'Doe' } }).first()
 
 // Exists
-const userFound: boolean = await photon.user.findUnique('bobs-id').exists()
-const foundAtLeastOneUser: boolean = await photon.user
-  .findMany({ email: { containsInsensitive: '@gmail.com' } })
-  .exists()
+const userFound: boolean = await photon.user.find('bobs-id').exists()
+const foundAtLeastOneUser: boolean = await photon.user.findMany({ email: { containsInsensitive: '@gmail.com' } }).exists()
 
 // Simple aggregation short
 // TODO more examples
 const deletedCount: number = await photon.user.delete().count()
-uts
+```
+
+## Field-level Primary Key constraint
+
+Different unique constraints will change Photon's `where` blocks:
```

Comment:
`where` block is the thing inside `find(...)`?

Reply:
yep